tests/acceptance/tests/Account/GeoBlockingNonShippableCountry.spec.ts

```diff
@@ -0,0 +1,188 @@
+
+        await test.step('Customer submits the registration form successfully with a shippable country', async () => {
+            await StorefrontAccountLogin.shippingAddressCountryInput.selectOption({ label: shippableCountry.name });
+            await ShopCustomer.expects(StorefrontAccountLogin.shippingAddressStateInput).toBeVisible();
+            await StorefrontAccountLogin.shippingAddressFirstNameInput.fill(defaultRegistrationData.firstName);
+            await StorefrontAccountLogin.shippingAddressLastNameInput.fill(defaultRegistrationData.lastName);
+            await StorefrontAccountLogin.shippingAddressStreetAddressInput.fill(defaultRegistrationData.street);
+            await StorefrontAccountLogin.shippingAddressPostalCodeInput.fill(defaultRegistrationData.postalCode);
+            await StorefrontAccountLogin.shippingAddressCityInput.fill(defaultRegistrationData.city);
+            await StorefrontAccountLogin.registerButton.click();
+
+            await ShopCustomer.expects(StorefrontAccount.headline).toBeVisible();
+            await deleteRegisteredUser(AdminApiContext, customer.email);
+        });
+    }
+);
+
+test(
+    'Customers is not able to set new shipping address with a non-shippable country.',
+    { tag: '@Account @Address' },
+    async ({
+        IdProvider,
+        ShopCustomer,
+        TestDataService,
+        DefaultSalesChannel,
+        StorefrontAccountLogin,
+        Login,
+        Logout,
+        AddNewAddress,
+        StorefrontAccount,
+        StorefrontAccountAddresses,
+        Register,
+        InstanceMeta,
+    }) => {
+        const nonShippableCountry = await TestDataService.createCountry({
+            shippingAvailable: false,
+            states: [
+                {
+                    name: 'California',
+                    shortCode: 'CA',
+                },
+            ],
+        });
+        await TestDataService.assignSalesChannelCountry(DefaultSalesChannel.salesChannel.id, nonShippableCountry.id);
+        const shippableCountry = await TestDataService.createCountry({
+            states: [
+                {
+                    name: 'California',
+                    shortCode: 'CA',
+                },
+            ],
+        });
+        await TestDataService.assignSalesChannelCountry(DefaultSalesChannel.salesChannel.id, shippableCountry.id);
+
+        const defaultRegistrationData: RegistrationData = {
+            isCommercial: false,
+            isGuest: false,
+            salutation: 'Mr.',
+            firstName: 'Jeff',
+            lastName: 'Goldblum',
+            email: `${IdProvider.getIdPair().uuid}@test.com`,
+            password: 'shopware',
+            street: 'Ebbinghof 10',
+            city: 'Schöppingen',
+            country: `${nonShippableCountry.name} (Delivery not possible)`,
+            postalCode: '48624',
+            company: 'shopware',
+            department: 'Operations',
+            vatRegNo: 'DE1234567890',
+        };
```

Comment:
You don't need them. Instead use only necessary register data like  this:
```suggestion
const customer = { email: `${IdProvider.getIdPair().uuid}@test.com`, country: `${nonShippableCountry.name} (Delivery not possible)` };
```